src/NodeApi/JSReference.cs

```diff
@@ -28,8 +28,15 @@ public class JSReference : IDisposable
     private readonly napi_env _env;
     private readonly JSRuntimeContext? _context;
 
-    public bool IsWeak { get; private set; }
-
+    /// <summary>
+    /// Creates a new instance of a <see cref="JSReference"/> that holds a strong or weak
+    /// reference to a JS value.
+    /// </summary>
+    /// <param name="value">The JavaScript value to reference.</param>
+    /// <param name="isWeak">True if the reference will be "weak", meaning the reference does not
+    /// prevent the value from being released and garbage-collected. The default is false,
```

Comment:
Should we say that GC in this context is JS GC, not .Net GC?

Reply:
Updated.